src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/ReducibleAnnotation.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.hellbender.tools.walkers.annotator;
+
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.utils.genotyper.PerReadAlleleLikelihoodMap;
+
+import java.util.Map;
+
+/**
+ * An interface for annotations that are calculated using raw data across samples, rather than the median (or median of median) of samples values
+ */
```

Comment:
Make note of the missing functionality in this interface compared to GATK3, and create a ticket to track this unported code.